client/src/components/_common/Pill/Pill.module.scss

```diff
@@ -1,11 +1,13 @@
-.pill {
-  padding: 0 1em; /* buffer retained on 2+ digit badge, without increase on 1 digit badge */
-  min-width: 1.3em;  /* ~18px design * 1.2 design-to-app ratio (rounded) */
+/* TODO: Consider sharing code between (History)Badge and Pill */
+.root {
+  font-size: 0.75rem; /* 12px (10px design * 1.2 design-to-app ratio) */
+  font-weight: 500;
+  padding: 0.2em 0.5em; /* aim for ~23px height (19px design * 1.2 design-to-app ratio) */
+  display: inline-block; /* FAQ: Supports `min-width` */
+  /* min-width: ____; *//* FAQ: Set only as needed (not always needed) */
   text-align: center;
-  border-radius: 3.6px;
+  border-radius: 3px;
   color: var(--global-color-primary--xx-light);
-  margin-left: auto;
-  margin-right: 2em;
```

Comment:
These came from `HistoryBadge`, but did not seem relevant for `Pill`.

Reply:
Ah, ok. The Tags mockup in XD looks like it had some extra space but if you don't think it needs it then that's cool too.

Reply:
[Design system](https://xd.adobe.com/view/bdd30952-4f4a-4e8d-5a42-29670859679c-cbe2/specs/), right? Nah, that's just the icons being placed apart for visibility.

_**Use Cases:** Context will decide spacing around them. Also, when I look through instances in [Core portal](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/grid/), the spacing around them should be attributed table cell spacing. The screenshots in [FP-451](https://jira.tacc.utexas.edu/browse/FP-451) do not have side-by-side pills, and if they did, that margin would be context-specific._

_**Architecture:** The pills are self-contained elements, spacing around them is for their container to decide (treating the pills as content) ([separation of container and content](https://confluence.tacc.utexas.edu/display/UP/UI+-+CSS+Style+Guide+-+Architecture#UI-CSSStyleGuide-Architecture-SeparationofContainerandContent))._